nextcord/application_command.py

```diff
@@ -367,6 +367,10 @@ def payload(self) -> dict:
         if self.channel_types:
             # noinspection PyUnresolvedReferences
             ret["channel_types"] = [channel_type.value for channel_type in self.channel_types]
+            ret[
+                "channel_types"
+            ].sort()  # When reading application commands from Discord, they return the channel types
+            #  sorted. To prevent needless syncing and allow lazy loading, we need to sort them as well.
```

Comment:
Better to put comments above for readability, can use `sorted` as a more "functional" and concise approach
```suggestion
            # When reading application commands from Discord, they return the channel types sorted.
            # To prevent needless syncing and allow lazy loading, we need to sort them as well.
            ret["channel_types"] = sorted([channel_type.value for channel_type in self.channel_types])
```